stringCompression.py

```diff
@@ -0,0 +1,32 @@
+def compress_string(input_string):
+    compressed = []
+    count = 1
+
+    for i in range(1, len(input_string)):
+        if input_string[i] == input_string[i - 1]:
+            count += 1
+        else:
+            compressed.append(input_string[i - 1] + str(count))
+            count = 1
+
+    #adds the character and its count to the end of the compressed list
+    compressed.append(input_string[-1] + str(count))
+
+    #combines entries in list into a single string
+    compressed_str = ''.join(compressed)
+
+    if len(compressed_str) < len(input_string):
+        return compressed_str
+    else:
+        return input_string
```

Comment:
Because you're returning in both the `if` and `else` cases with no extra computation, you can do this cleanly in one line using a ternary.
```suggestion
    return compressed_str if len(compressed_str) < len(input_string) else input_string
```